### ISSUE TITLE
Permission denied when following `BUILD.rst`

### ISSUE THREAD

**Issue by alexprengere** [squeaky-pl/portable-pypy#88]
I am trying to follow the `BUILD.rst` instructions, but I have an issue:
```bash
$ docker run --rm `ABI=64 ./runopt.sh` portable-pypy ./build_deps 
/bin/bash: ./build_deps: Permission denied
```
 I run this from a fresh clone, on a Fedora 22 (and docker 1.8, which I think is a bit old).
I hope I did not miss something obvious. Do you think this may be related to the docker version?

**Comment by squeaky-pl** [comment]
This doesnt happen on my machine, what does `docker run --rm `ABI=64 ./runopt.sh` portable-pypy ls -lha ./build_deps` say

**Comment by alexprengere** [comment]
`ls: cannot access ./build_deps: Permission denied`

**Comment by squeaky-pl** [comment]
Looks like the USER id inside docker doesnt have access to files inside `src64` and `prefix64` on your host that are mounted as volumes

can you `ls -lha src64 prefix64` and `docker run --rm portable-pypy id -a`

**Comment by squeaky-pl** [comment]
it's possible that docker volume support was not working for what I am doing inside `runopt.sh` in 1.8 version.

**Comment by alexprengere** [comment]
I think you right, I will set up a couple of Ubuntu/Fedora VMs to test this on more recent setups.
```
$ ls -lha src64
drwxr-xr-x. 2 root    root    4.0K Feb 14 05:20 ./
drwxr-xr-x. 6 osboxes osboxes 4.0K Feb 14 05:20 ../
-rwxr-xr-x. 1 root    root       0 Feb 14 05:20 build*
-rwxr-xr-x. 1 root    root       0 Feb 14 05:20 build_deps*
-rwxr-xr-x. 1 root    root       0 Feb 14 05:20 _curses_build.py.2.patch*
-rwxr-xr-x. 1 root    root       0 Feb 14 05:20 _curses_build.py.patch*
-rwxr-xr-x. 1 root    root       0 Feb 14 05:20 make_portable*
-rwxr-xr-x. 1 root    root       0 Feb 14 05:20 package*
-rwxr-xr-x. 1 root    root       0 Feb 14 05:20 platform_linux.patch*
-rwxr-xr-x. 1 root    root       0 Feb 14 05:20 socket.append.py*
-rwxr-xr-x. 1 root    root       0 Feb 14 05:20 ssl3.py.patch*
-rwxr-xr-x. 1 root    root       0 Feb 14 05:20 ssl.py.patch*
-rwxr-xr-x. 1 root    root       0 Feb 14 05:20 subprocess3.py.patch*
-rwxr-xr-x. 1 root    root       0 Feb 14 05:20 subprocess.py.patch*
-rwxr-xr-x. 1 root    root       0 Feb 14 05:20 sysconfig_pypy3.py.patch*
-rwxr-xr-x. 1 root    root       0 Feb 14 05:20 sysconfig_pypy.py.patch*
-rwxr-xr-x. 1 root    root       0 Feb 14 05:20 _tkinter_app.py.patch*
-rwxr-xr-x. 1 root    root       0 Feb 14 05:20 version.py*
-rwxr-xr-x. 1 root    root       0 Feb 14 05:20 virtualenv-pypy*

$ ls -lha prefix64   # empty
drwxr-xr-x. 2 root    root    4.0K Feb 14 05:20 ./
drwxr-xr-x. 6 osboxes osboxes 4.0K Feb 14 05:20 ../

$ docker run --rm portable-pypy id -a
uid=0(root) gid=0(root) groups=0(root)
```

**Comment by alexprengere** [comment]
So I tried on a fresh Ubuntu and I can confirm I do not have this issue with docker 18. But now I am hit by #56 :wink:.

**Comment by alexprengere** [comment]
For the record, I tested also this on Fedora 29 (latest stable), which has docker 1.13 by default. I got the same error. Then I followed [this](https://docs.docker.com/install/linux/docker-ce/fedora) to install the latest docker (18.09.2), then it works. Also I did not encounter #56 as I did with the latest Ubuntu.

Long story short for people getting a similar issue: use a recent docker version on Fedora/CentOS.
